inventory-service/src/test/java/com/petclinic/inventoryservice/businesslayer/ProductInventoryServiceUnitTest.java

```diff
@@ -46,14 +45,171 @@ class ProductInventoryServiceUnitTest {
             .productPrice(100.00)
             .productQuantity(10)
             .build();
-
+    Product product = Product.builder()
+            .productId("12345")
+            .inventoryId("1")
+            .productName("Benzodiazepines")
+            .productDescription("Sedative Medication")
+            .productPrice(100.00)
+            .productQuantity(10)
+            .build();
     Inventory inventory = Inventory.builder()
             .id("1")
             .inventoryId("1")
             .inventoryType(InventoryType.internal)
             .inventoryDescription("Medication for procedures")
             .build();
 
+   /* @Test
+    void getAllProductsByInventoryId_withValidFields_shouldSucceed(){
+        String inventoryId = "1";
+
+        when(productRepository
+                .findAllProductsByInventoryIdAndProductNameAndProductPriceAndProductQuantity(
+                        inventoryId,
+                        null,
+                        null,
+                        null))
+                .thenReturn(Flux.just(product));
+
+        Flux<ProductResponseDTO> productResponseDTOFlux = productInventoryService
+                .getProductsInInventoryByInventoryIdAndProductsField(
+                        inventoryId,
+                        null,
+                        null,
+                        null);
+
+        StepVerifier
+                .create(productResponseDTOFlux)
+                .expectNextCount(2)
+                .verifyComplete();
+    }*/
+
```

Comment:
Remove any tests that are not going to be used, its best to not push commented out code to main